docs/references/chrome-extension/create-clerk-client.mdx

```diff
@@ -3,131 +3,161 @@ title: '`createClerkClient()`'
-
-
-```tsx {{ filename: 'src/tabs/background-worker-demo.tsx' }}
-import * as React from 'react';
-
-export default function NewTab() {
-  const [token, setToken] = React.useState<string | null>(null);
-
-  const getToken = async (e: React.MouseEvent<HTMLButtonElement>) => {
-    e.preventDefault();
-
-    // send a message to the background service worker
-    chrome.runtime.sendMessage({ greeting: 'get-token' }, response => {
-      console.log('MESSAGE RESPONSE', JSON.stringify(response));
-      setToken(response.token);
-    });
-  };
-
-  return (
-    <div>
-      <p>Clerk Background Worker demo</p>
-      <div className="App">
-        <p>This new tab simluates a content page where you might want to access user information, or make a request to your backend server and include a user token in the request.</p>
-        <p>Make sure that you are signed into the extension. You can have the popup closed.</p>
-        <button
-          type='button'
-          onClick={getToken}
-          className='button invert'
-        >
-          Get Token (Service Worker) asdfasdfds
-        </button>
-        {token && <p>Token: {token}</p>}
-      </div>
-
-    </div>
-  )
-}
-```
-
-## Add a button to the extension to open the new tab
-
-Add a button to your Chrome Extension to open that page you created as a new tab. This can be added anywhere in your extension. The below example would place the button on the home page of the extension if you followed the [React Router guide](/docs/references/chrome-extension/add-react-router) for Chrome Extensions.
-
-```tsx {{ filename: 'src/popup/routes/index.tsx' }}
-  export const Index = () => {
+    // This example sends the token back to the content script
+    // but you could also use the token to perform actions on behalf of the user
+    getToken().then((token) => sendResponse({ token }))
+    return true // REQUIRED: Indicates that the listener responds asynchronously.
+  })
+  ```
+
+  ### Create the tab with the content script
+
+  1. In the `src/` directory, create the `tabs/` directory.
+  1. In the `tabs/` directory, create the `background-worker-demo.html` file.
+  1. In the `background-worker-demo.html` file, paste the following code to create a basic HTML file that will house the React component for the content script.
+     ```html {{ filename: 'src/tabs/background-worker-demo.html' }}
+     <!doctype html>
+     <html>
+       <head>
+         <title>Clerk Background Worker Demo</title>
+         <meta charset="utf-8" />
+         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+       </head>
+
+       <body></body>
+     </html>
+     ```
+  1. In the `tabs/` directory, create the `background-work-demo.tsx` file.
```

Comment:
1. In the `tabs/` directory, create the `background-worker-demo.tsx` file.

Reply:
```suggestion
  1. In the `tabs/` directory, create the `background-worker-demo.tsx` file.
```